app/models/related_instruction.rb

```diff
@@ -0,0 +1,3 @@
+class RelatedInstruction < ApplicationRecord
+  belongs_to :occupation_standard
```

Comment:
@jenlu33 can you also make sort order unique within the occupation standard? See Fer's PR #37 in the `app/models/competency.rb` for similar set up.